Cargo.toml

```diff
@@ -6,9 +6,12 @@ edition = "2021"
 [dependencies]
 hex = "0.4.3"
 u256 = { package = "primitive-types", version = "0.12.1" }
+k256 = "0.13.3"
 zkevm_opcode_defs = { git = "https://github.com/matter-labs/era-zkevm_opcode_defs.git", branch = "v1.5.1" }
-zk_evm_abstractions = { git = "https://github.com/matter-labs/era-zk_evm_abstractions.git", branch = "v1.5.1" }
-# zk_evm_abstractions = { path = "../era-zk_evm_abstractions" }
 rocksdb = "0.21.0"
 thiserror = "1.0.61"
+sha2 = "0.11.0-pre.0"
+sha3 = "0.11.0-pre.0"
```

Comment:
I'm not sure using a pre-release is a good idea.

Reply:
Good catch, can we change this @juan518munoz ?

Reply:
We are trying to change them, but we are having some issues when getting the intermediate state of the hasher. We were using the crypto commons crate for that, but the function to do so is only available on the pre release version.